events/search_built.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+import "github.com/ONSdigital/go-ns/avro"
+
+// SearchBuilt contains data related to a search that has just been built.
+type SearchBuilt struct {
+	InstanceID    string `avro:"instance_id"`
+	DimensionName string `avro:"dimension_name"`
+}
+
+var searchIndexBuilt = `{
+  "type": "record",
+  "name": "search-index-built",
+  "fields": [
+    {"name": "instance_id", "type": "string", "default": ""},
+    {"name": "dimension_name", "type": "string", "default": ""}
+  ]
+}`
+
+// SearchIndexBuiltSchema is the Avro schema for each dimension hierarchy successfuly sent to elastic
+var SearchIndexBuiltSchema *avro.Schema = &avro.Schema{
```

Comment:
Just a note: you don't explicitly need to declare `*avro.Schema` - it will infer the type. 

```
var SearchIndexBuiltSchema = &avro.Schema{
```

Not sure if there are any standards for this to suggest one to use, and its fine either way I think